python-threatexchange/threatexchange/cli/hash_cmd.py

```diff
@@ -93,16 +93,16 @@ def _parse_input(
     def execute(self, api: ThreatExchangeAPI, dataset: Dataset) -> None:
 
         all_signal_types = dataset.load_cache(
-            s
+            s()
```

Comment:
This shouldn't be required if the target function is a classmethod. Why this change?

Reply:
SignalType::load called in dataset is not a classmethod https://github.com/facebook/ThreatExchange/blob/5acf1bf5fdb47873c83f71bb785b14f3583805c2/python-threatexchange/threatexchange/signal_type/signal_base.py#L114